config/routes.rb

```diff
@@ -0,0 +1,77 @@
+Rails.application.routes.draw do
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+
+  # You can have the root of your site routed with "root"
+  # root 'welcome#index'
+
+  # Example of regular route:
+  #   get 'products/:id' => 'catalog#view'
+
+  # set root routing
+  root 'tasks#index'
+
+  # set Task routes
+  get '/tasks'              => 'tasks#index', as: 'tasks' # display a list of all tasks
+  get '/tasks/new'          => 'tasks#new'  # return an HTML form for creating a new task
+  post '/tasks'             => 'tasks#create' # create a new task
+  get '/tasks/:id'          => 'tasks#show', as: 'show' # display a specific task
+  get '/tasks/:id/edit'     => 'tasks#edit', as: 'task' # return an HTML form for editing a task
+  patch '/tasks/:id/edit'   => 'tasks#update' # update a specific task
+  delete '/tasks/:id'       => 'tasks#delete', as: 'delete'  # delete a specific task
+  post 'tasks/:id/'         => 'tasks#complete'
+
+
+  # set People Routes
+  get '/people'             => 'people#index', as: 'people'
+  get '/person/:id/tasks'    => 'people#show', as: 'person_detail'
+
```

Comment:
You renamed some of your routes - thats pretty rad and gives a more descriptive URL than 'show'.  I was able to go to person/1/tasks and see all of Howard's tasks.  It would be helpful if you had some way to link to that on your homepage, like a navbar, otherwise someone visiting your page may not know that that awesome feature is available.